crossword-xml-uploader/src/main/scala/com/gu/crossword/crosswords/CrosswordConfigRetriever.scala

```diff
@@ -18,6 +18,10 @@ trait S3CrosswordConfigRetriever extends CrosswordConfigRetriever {
     val config = loadConfig(stage)
 
     val crosswordMicroAppUrl = Option(config.getProperty("crosswordmicroapp.url")) getOrElse sys.error("'crosswordmicroapp.url' property missing.")
+
+    // Fail safe in case crosswordV2Url is not set
+    val crosswordV2Url = Option(config.getProperty("crosswordv2.url")) getOrElse "https://crossword-v2-url"
```

Comment:
may be more idiomatic to leave as optional and handle the option before calling doV2Upload (or inside that function)

Reply:
Something about returning an option here made me uncomfortable. I think perhaps having to then unwind it when we switch over. This is properly an option though as we want to account for the case it isn't present (and the default has other impacts we might want to avoid).